frontend/AdminPanel/TestCreator.svelte

```diff
@@ -39,6 +40,16 @@
                     bind:value={newTest.test_name}
                 />
             </div>
+            <div class="form-group mb-2">
+                <label for="" class="form-label">Plugin</label>
+                <select id="" class="form-select" bind:value={newTest.plugin_name}>
+                    {#each Object.keys(AVAILABLE_PLUGINS) as plugin}
+                        <option value={plugin}
+                            >{plugin}</option
+                        >
```

Comment:
should be on one line?

Reply:
For inline elements like that formatting it like this avoids inserting extra spaces into markup, the last `>` is there to line up with opening `<option>`